lib/cloudregister/registerutils.py

```diff
@@ -327,6 +344,8 @@ def fetch_smt_data(cfg, proxies):
                         if srv == region_servers[-1]:
                             logging.error('\tAll servers reported an error')
                 except requests.exceptions.RequestException:
+                    if quiet:
+                        continue
```

Comment:
with this we are not showing the logging errors, I understand that is intentional
shouldn't we show the logging errors instead of no info ?

IMHO the logging of the error should show up unconditionally

Reply:
We do not want to pollute the log file every time zypper runs. I was trying to find a better way to skip trying to access the region servers via IPv6 but have not yet has a good idea. Ultimately if there is a total failure, i.e. we cannot fins any regionsrv that responds then we get the message from line 345 and during debugging we known what happened. The detailed messages about the region servers are in the log at the beginning when we first register the system. I see no point in polluting the log file with these messages over and over again.

Reply:
>  I was trying to find a better way to skip trying to access the region servers via IPv6 

Maybe check if the srv is IPv6 and skip it ?
```python
if isinstance(ipaddress.ip_address(srv), ipaddress.IPv6Address):
    continue
```
could have an extra param too, `skip_ipv6` (similar to `quiet`)